examples/serveless/README.md

```diff
@@ -0,0 +1,79 @@
+Alibaba Cloud terraform example for kubernetes cluster
+======================================================
+
+A terraform example to launching a serveless kubernetes cluster in alibaba cloud.
+
+These types of the module resource are supported:
+
+- [VPC](https://www.terraform.io/docs/providers/alicloud/r/vpc.html)
+- [Subnet](https://www.terraform.io/docs/providers/alicloud/r/vswitch.html)
+- [Serveless](https://www.terraform.io/docs/providers/alicloud/r/cs_serveless_kubernetes.html)
+
+
+Usage
+-----
+This example can specify the following arguments to create user-defined kuberntes cluster
+
+* alicloud_access_key: The Alicloud Access Key ID
+* alicloud_secret_key: The Alicloud Access Secret Key
+* region: The ID of region in which launching resources
+* serveless_cluster_name: The name  of serveless cluster
+* Other kubernetes cluster arguments
+
+**Note:** In order to avoid some needless error, you had better to set `new_nat_gateway` to `true`.
+Otherwise, you must you must ensure you specified vswitches can access internet before running the example.
+
+Planning phase
+
+    terraform plan
+
+Apply phase
+
+	terraform apply
```

Comment:
still need to format this part

Reply:
the format is correct, i double check it using markdown tools